src/writeData/subscriptions/components/SubscriptionCard.tsx

```diff
@@ -1,5 +1,6 @@
 // Libraries
 import React, {FC, useContext} from 'react'
+import moment from 'moment'
```

Comment:
I believe we've moved away from `moment` in the UI since it has been [deprecated](https://github.com/moment/moment). It looks like it's only available in package.json as a devDep.

Reply:
ah ok i'll change that out. I just searched moment and saw it imported without checking further